auctions/dutch-auction/project/src/interface.sw

```diff
@@ -0,0 +1,136 @@
+library;
+
+use ::data_structures::Auction;
+
+abi DutchAuction {
```

Comment:
I would split this into `Info` like the other apps to separate the getters and follow the project structure in those tests